host_vars/OCT4-SW-TORS/interfaces.yaml

```diff
@@ -106,9 +106,9 @@ interfaces:
       - 213
       - 250
       - 623
+      - 626
```

Comment:
`626` isn't in [this file](https://github.com/CCI-MOC/ansible-switches/blob/main/group_vars/all/vlans.yaml). Is this a user VLAN?

Reply:
Ah, it is - let me remove it!